### PR TITLE
Integration test action

### PR DESCRIPTION
**Pull request by benmor20**
GitHub action that declines merges to `true-head` which do not pass integration tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,2 @@
 # noodlemaps
 Better multi-stop path planning built on google maps
-
-hey ben get on linux
```
Comment (berwinlan) [inline README.md:4]: i disapprove >:(

**Review by berwinlan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hnvakil) [reply README.md:4]: I approve this message I mean review

**Review by hnvakil — APPROVED** [review]
This all looks good

**Comment by hnvakil** [comment]
Looks good! Merging

**Pull request merged**